src/main/java/uk/gov/digital/ho/hocs/aws/SQSConfiguration.java

```diff
@@ -19,7 +19,7 @@
     @Bean(name = "sqsClient")
     public AmazonSQS sqsClient(@Value("${case.creator.sqs.access-key}") String accessKey,
                                @Value("${case.creator.sqs.secret-key}") String secretKey,
```

Comment:
The environment variable set in kube is CASE_CREATOR_SQS_SECRET_KEY, which equate to case.creator.sqs.secret.key
```suggestion
                               @Value("${case.creator.sqs.secret.key}") String secretKey,
```

Reply:
The naming conventions comes from the spring boot properties. The way spring boot works now is to match a dot with a hierarchy level. So if a word is in camel case e.g secretKey then in properties it would be secret-key.

Reply:
Just had a look at the docs and it seems both ways are supported. All good.

Reply:
Just had a look at the docs and it seems both ways are supported. All good.